README.md

```diff
@@ -2,11 +2,11 @@
 - Implementation of OpenID4VP protocols in Swift.
 
 ## Functionalities
-- Process the Authorization Request of the verifier received from the Wallet.
-- Validate and return the valid Presentation Definition to the wallet.
-- Receives the list of verifiable credentials from the wallet.
-- Constructs the verifiable presentation and send it to wallet for proof generation.
-- Receives the signed Verifiable presentation and sends a POST request to the URL specified in the verifier request. 
+- Decode and parse the Verifier's encoded Authorization Request received from the Wallet.
+- Authenticates the Verifier using the received clientId and returns the valid Presentation Definition to the Wallet.
+- Receives the list of verifiable credentials(VC's) from the Wallet which are selected by the Wallet end user based on the credentials requested as part of Verifier Authorization request.
```

Comment:
can we change this to `Receives the list of verifiable credentials(VC's) from the Wallet which are selected by the end user based on the credentials requested as part of Verifier Authorization request.`